.vsts.pipelines/jobs/ci-linux_bootstrap.yml

```diff
@@ -0,0 +1,96 @@
+parameters:
+  job: null
+  matrix:
+    Production: {}
+  pool:
+    name: Hosted Ubuntu 1604
+  imageName: null
+  dependsOn: ''
+
+jobs:
+- job: ${{ parameters.job }}
+  dependsOn: ${{ parameters.dependsOn }}
+  strategy:
+    matrix: ${{ parameters.matrix }}
+  pool: ${{ parameters.pool }}
+  timeoutInMinutes: 270
+  variables:
+    # Prefix to distinguish artifacts from different legs.
+    artifactName: ${{ format('{0} $(type)', parameters.job) }}
+    # Use ":z" to set selinux flag for sharing in build-owned root dir. https://docs.docker.com/storage/bind-mounts/#configure-the-selinux-label
+    docker.agentSrc.map: -v $(Build.SourcesDirectory):/agentSrc:z
+    docker.agentSrc.work: -w /agentSrc
+    docker.drop.map: -v $(dropDirectory):/drop:z
+    docker.logs.map: -v $(dropDirectory)/logs:/logs:z
+    docker.root.map: -v $(rootDirectory):/root:z
+    docker.run: docker run --rm
+    docker.bst.map: -v $(System.ArtifactsDirectory)/boostrap_root:/bst:z
+    docker.bst.work: -w /bst
+    dropDirectory: $(stagingDirectory)/drop
+    imageName: ${{ parameters.imageName }}
+    rootDirectory: $(Build.SourcesDirectory)/..
+    stagingDirectory: $(rootDirectory)/bst/staging
+    tarballName: tarball_$(Build.BuildId)
+    SOURCE_BUILD_SKIP_SUBMODULE_CHECK: true
```

Comment:
I don't think this is being used.  It only applies to a production build.

Reply:
Do you mean `SOURCE_BUILD_SKIP_SUBMODULE_CHECK`?

Reply:
Yes.  I don't think that will be used, since they're all tarball builds.